docs/reference/api/drivers/SAI.md

```diff
@@ -0,0 +1,52 @@
+## Serial Audio Interface (SAI)
+
+The Serial Audio Interface (SAI) provides a class to communicate with audio devices. SAI is a broader feature that encompasses audio interfaces such as I2S.
+
+SAI uses a four wire serial protocol consisting of:
+  * Master Clock Pin  (mclk)
+  * Data Pin          (sd)
+  * Bit Clock pin     (bclk)
+  * Word Clock Pin    (wclk)
+
```

Comment:
"Pin" should be capitalized or lowercase consistently (the first two are capitalized, the next two are lowercase).  I'm changing all to lower case.